src/main/java/com/mongodb/kafka/connect/source/MongoSourceTask.java

```diff
@@ -192,6 +192,12 @@ public void start(final Map<String, String> props) {
             getTopicNameFromNamespace(
                 prefix, changeStreamDocument.getDocument("ns", new BsonDocument()));
 
+        if (topicName.isEmpty()) {
```

Comment:
Should we also add a test for this logic?

Reply:
I spun this work out of #27 - where I noticed some areas that needed fixes but didn't want to conflate that PR.

The topic name is missing when Invalidate events are published (dropped collections can cause this). The test speed ups in #26 highlighted this issue as it changes how the tests are processed and this ticket was added before processing on #26 then #27.

Reply:
Ah ok, great.